flask_webpackext/project.py

```diff
@@ -56,6 +56,16 @@ def flask_config():
     }
 
 
+def flask_allowed_copy_paths():
+    """Get the allowed copy paths from the Flask application."""
+    return [
+        current_app.instance_path,
+        current_webpack.project.path,
+        current_app.static_folder,
+        current_app.config["WEBPACKEXT_PROJECT_DISTDIR"],
```

Comment:
note: in my setup, they're basically just subdirectories of the `instance_path`.

if there is a chance that these could actually point to different paths, i would still keep it.
a few redundant entries in this list may look a bit weird in the average setup, but that's about it; performance won't be impacted by a so few extra entries.